pkg/kube/watcher.go

```diff
@@ -57,7 +57,8 @@ func (e *EventWatcher) OnAdd(obj interface{}) {
 }
 
 func (e *EventWatcher) OnUpdate(oldObj, newObj interface{}) {
-	// Ignore updates
+	event := newObj.(*corev1.Event)
+	e.onEvent(event)
```

Comment:
This should be on other update, carefully thought. Not all sinks are compatible with this kind of update flow.

Reply:
I just tried to restore what was before. Anyway, there is already an open PR for such configurable functionality #126.

@mustafaakin please review.